pyperf/pyperf_run

```diff
@@ -245,15 +252,18 @@ done
 
 if [ $to_pbench -eq 0 ]; then
 	rm -rf pyperformance
-	git clone https://github.com/python/pyperformance
-	if [ $? -ne 0 ]; then
-		exit_out "Cloning of https://github.com/python/pyperformance failed." 1
+	#
+	# Install pip/pip3
+	#
+	python3 -m ensurepip --upgrade
+	if [[ $? -ne 0 ]]; then
+		exit_out "Failed: python3 -m ensurepip --upgrade" 1
```

Comment:
After digging around, RHEL and Ubuntu both use a `python3-pip` package.

Considering [this](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/installing_and_using_dynamic_programming_languages/assembly_installing-and-using-python_installing-and-using-dynamic-programming-languages#proc_installing-additional-python-3-packages_assembly_installing-and-using-python) product doc, we should be specifying `python3-pip` as a dependency.